contracts/middle-layer/resource-mirror/PermissionHub.sol

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "../../interface/IPermissionHub.sol";
+import "../../interface/ICrossChain.sol";
+import "./CmnHub.sol";
+
+contract PermissionHub is PermissionStorage, CmnHub, IPermissionHub {
+    constructor() {
+        _disableInitializers();
+    }
+
+    /*----------------- initializer -----------------*/
+    function initialize(address _ERC721_token, address _additional) public initializer {
+        __cmn_hub_init_unchained(_ERC721_token, _additional);
+
+        channelId = PERMISSION_CHANNEL_ID;
+    }
+
+    function initializeV2() public reinitializer(2) {
```

Comment:
This is a newly deployed contract, no need to have `initializeV2`

Reply:
In order to keep the code completely consistent with the BucketHub, GroupHub and ObjectHub, this makes it easier to compare the code difference